handbook/company/communications.md

```diff
@@ -111,11 +111,15 @@ It is also common practice to block out time for focused work.
 
 <blockquote puropse="large-quote">In an all-remote company, "face time" matters. Remember: even if someone's calendar is open, they have other work to do. Limiting (or batching up) internal meetings can enable longer, uninterrupted stretches of deep work.</blockquote>
 
-
 ### Modifying an event organized by someone else
 To edit an event where someone else at Fleet is the organizer, you can first subscribe to their calendar in Google Calendar and then edit the event on their calendar.  Your edits will automatically apply to all attendees.
 This works because every Fleetie grants edit access to everyone else at Fleet as part of onboarding.
 
+### Shared calendars
+At Fleet, our shared team calendars are more than just schedules; they are the heartbeats of our operations. Acting as the primary source for team rituals, they facilitate the smooth execution of each sprint.
```

Comment:
Let's cut the first sentence (less surface area).  Victory is when we've cut down the words on this page like 50%